_posts/2022-01-14-quorn-a-story-about-single-cell-protein.md

```diff
@@ -0,0 +1,197 @@
+
+Maybe I'll ask Quorn for some, or make my own. The original patents for this bioprocess have already expired - though it would be difficult to replace Marlow Foods' 35 years of operating experience. 
+
+##### The Protein Crisis 
+
+The original goal at inception, to prevent protein malnutrition, was never really met - because thankfully, and generally speaking - the 'Protein Crisis' neve happened agricultural productivity kept up with population growth through the 1960s, 70s, 80s, and beyond. This was mostly due to the *Green Revolution* consisting of better crop strains, fertilizer use, and the like. SCP could never compete economically with plant proteins. 
+
+But I can't fault the authors of the original UN paper: in 1967 we hadn't even discovered RuBisCO, a key enzyme behind photosynthesis. This discovery wouldn't be made until well into the 1980s [12]. Before that, it might have been safe to assume that plants simply assimilate nitrogen to create protein, so we could bypass the process altogether by adding sugar and nitrogen into a bioreactor.
+
+As it turns out, protein synthesis and photosynthesis are directly coupled. Photosynthesis relies on the action of enzymes, especially RuBisCO, the massive enzyme used in carbon fixation and possibly the most abundant enzyme on earth. So yes, the nitrogen applied to soil is assimilated into protein, but that protein also enables photosynthesis and the formation of carbohydrates. It's impossible to compare Quorn to plant protein, especially since its primary feedstock - glucose - is derived from plants. 
+
+##### Space Travel and the Apocalypse 
+
+Quorn and SCP in general might be viable technologies in either an apocalyptic scenario - or space travel - as part of a larger system of chemical processes and bioreactors used to sustain human life. 
+
+Let's say an asteroid - on par with the one that wiped out the dinosaurs - impacted earth and the dust cloud shut down photosynthesis for decades, worldwide. That, or we bomb ourselves into a nuclear winter. In either scenario, SCP might be good insurance against a world without photosynthesis. Nuclear, geothermal or wind power would be required to fix carbon and SCP bioreactors might be our only food source once stockpiles run dry. 
+
+Or, let's say we want to colonize space. The same process might be viable as part of a life support system that closes the loop on human sustenance: exhaled CO2, water, and excreted feces and urine are directly recycled into food with energy as an input. This is a huge oversimplification, but at least it's thermodynamically feasible and may require less space or energy than an equivalent plant based farm. I'll explore this in a future post. 
```

Comment:
"may require less space" is interesting. You could have it running through pipes in the station, instead of a large room with overhead lights.

Reply:
The whole past month of reading SCP makes we want to do a thought experiment / napkin math on an all-bioreactor system vs a hydroponic-type plant-based system. I'm going to start working on this as my next post after industrial koji pt 3.